base/ltsect.dtx

```diff
@@ -815,20 +815,67 @@
 %    \end{macrocode}
 %  \end{macro}
 %
-% \changes{v1.0c}{1994/05/20}{Correct setting of \cs{protect}.}
-% \changes{v1.0h}{1994/11/04}{Added \cs{protected@write} to
-%    \cs{addtocontents}.  ASAJ.}
+%
+%
+%
+%
+%  \begin{macro}{\om@command@gobbled,\som@command@gobbled}
```

Comment:
Didn't hold back and wondered why they used variable csname prefixes rather than a constant `@gobble`, like `\@gobble@om` or something similar.

Reply:
it could just be \@gobble@om, might be more uniform, didn't think of that